src/componenten/Login.js

```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react';
+import TopMenu from "./TopMenu";
+import {useHistory} from "react-router-dom";
+
+function Login() {
+    const [logIn, setLogIn] = useState(true);
```

Comment:
@jamiebloem: de initiele waarde van logIn staat op true..in de uitwerkingen staat het op "false"